packages/main/src/MessageStrip.hbs

```diff
@@ -1,15 +1,16 @@
-<div class="sapMMessageStrip {{type}}">
+<div class="{{classes.main}}">
 	{{#unless ctr.hideIcon}}
-		<ui5-icon class="sapMMessageStripIcon" src="{{icon}}"></ui5-icon>
+		<ui5-icon class="ui5-messagestrip-icon" src="{{icon}}"></ui5-icon>
 	{{/unless}}
 
-	<ui5-label class="{{classes.label}}" wrap="true">{{ctr._nodeText}}</ui5-label> 
+	<span class="{{classes.label}}">{{ctr._nodeText}}</span> 

```

Comment:
The ui5-label is not used as not needed or?

Reply:
yeah, we don't really need a label for there as it is just text - we don't need any of label's features, so no need to reuse it over there.